readthedocs/audit/models.py

```diff
@@ -68,19 +74,26 @@ class AuditLog(TimeStampedModel):
 
     # pylint: disable=too-many-instance-attributes
 
-    PAGEVIEW = 'pageview'
-    DOWNLOAD = 'download'
-    AUTHN = 'authentication'
-    AUTHN_FAILURE = 'authentication-failure'
-    LOGOUT = 'log-out'
-
-    CHOICES = (
-        (PAGEVIEW, 'Page view'),
-        (DOWNLOAD, 'Download'),
-        (AUTHN, 'Authentication'),
-        (AUTHN_FAILURE, 'Authentication failure'),
-        (LOGOUT, 'Log out'),
-    )
+    class Actions(models.TextChoices):
+        PAGEVIEW = "pageview", _("Page view")
+        DOWNLOAD = "download", _("Download")
+        AUTHN = "authentication", _("Authentication")
+        AUTHN_FAILURE = "authentication-failure", _("Authentication failure")
+        LOGOUT = "log-out", _("Log out")
+        INVITATION_SENT = "invitation-sent", _("Invitation sent")
+        INVITATION_REVOKED = "invitation-revoked", _("Invitation revoked")
+        INVITATION_ACCEPTED = "invitation-accepted", _("Invitation accepted")
+        INVITATION_DECLINED = "invitation-declined", _("Invitation declined")
+
+    PAGEVIEW = Actions.PAGEVIEW
+    DOWNLOAD = Actions.DOWNLOAD
+    AUTHN = Actions.AUTHN
+    AUTHN_FAILURE = Actions.AUTHN_FAILURE
+    LOGOUT = Actions.LOGOUT
+    INVITATION_SENT = Actions.INVITATION_SENT
+    INVITATION_REVOKED = Actions.INVITATION_REVOKED
+    INVITATION_ACCEPTED = Actions.INVITATION_ACCEPTED
+    INVITATION_DECLINED = Actions.INVITATION_DECLINED
```

Comment:
Set these here to avoid having to refactor every place where we use them, and it's also easier to write `AuditLog.PAGEVIEW` than `AuditLog.Actions.PAGEVIEW`. We could also move this outside the model, so we can just write `Actions.PAGEVIEW`, but that will require changing every usage too, so.